learning/tour-of-beam/backend/docker-compose.yml

```diff
@@ -17,12 +17,19 @@ version: "3"
 
 services:
   datastore:
-    build: internal/storage/image
+    build: integration_tests/emulators/datastore 
     volumes:
       - ${DATASTORE_EMULATOR_DATADIR}:/opt/data
     environment:
-      - DATASTORE_PROJECT_ID=project-test
+      - DATASTORE_PROJECT_ID
```

Comment:
no value for the environment variable

Reply:
it's a special syntax for [passing variables in container](https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers)